modules/builtin/src/content-types/single_choice.js

```diff
@@ -10,6 +10,23 @@ function render(data) {
     })
   }
 
+  if (data.isDropdown) {
+    // Render dropdown
```

Comment:
Is that comment really necessary? I'm a bit unsure about this, because it's not guaranteed the extensions module is enabled... But at the same time, it does make sense..

Reply:
I say, we can keep this for now

@allardy you know my opinion on this : We get rid of extension and builtin modules and we let the channels implement their render functions.

If a channel support dropdown, be it if not then display it in another way.

@samuelmasse has a basic implementation for this I believe.

Reply:
@allardy Anyway for me to check which modules are loaded from here? I can do a check and log something if extensions is disabled!